modules/cert-manager-issuer-types.adoc

```diff
@@ -28,6 +29,7 @@ The following table outlines the test coverage for each tested issuer type:
 | CA                                 | Fully Tested       | Ensures basic CA functionality.
 | Self-signed                          | Fully Tested       | Ensures basic self-signed functionality.
 | Vault | Fully Tested   | Limited to standard Vault setups due to infrastructure access constraints.
+| Venafi | Partially tested | Subject to provider-specific limitations.
```

Comment:
Just a small nit: should we capitalize the initial letters of `Partially tested` to keep consistency?